src/material/dialog/dialog.md

```diff
@@ -14,7 +14,7 @@ let dialogRef = dialog.open(UserProfileComponent, {
 ```
 
 The `MatDialogRef` provides a handle on the opened dialog. It can be used to close the dialog and to
-receive notification when the dialog has been closed.
+receive notification when the dialog has been closed. Any notification Observables will close with the dialog.
```

Comment:
Not something you added, but we might as well fix it. The "receive notification" should be "receive notifications".

Also I think the correct terminology is that the observables will be "completed" when the dialog is closed, not that they'll be "closed".

Reply:
You're right, the terminology was indeed not correct. I was thinking of Subscriptions instead of Observables.
I fixed both, but had to rephrase slightly.